src/components/MessagesList/MessagesList.vue

```diff
@@ -51,6 +51,13 @@ get the messagesList array and loop through the list to generate the messages.
 				type="messages"
 				:count="15" />
 		</template>
+		<transition name="fade">
+			<button v-show="!isScrolledToBottom"
+				class="scroll-to-bottom"
+				@click="scrollToBottom">
+				<ChevronDown :size="24" fill-color="#fff" />
```

Comment:
Tooltip says `Chevron Down icon`.
We have the same issue in quite some other places (overflowing video list is `Chevron Right icon` etc.
I guess we need to set the title ourselves